apps/govquests-api/govquests/notifications/lib/notifications.rb

```diff
@@ -0,0 +1,18 @@
+require "infra"
+require_relative "notifications/commands"
+require_relative "notifications/events"
+require_relative "notifications/template_commands"
+require_relative "notifications/template_events"
+require_relative "notifications/on_notification_commands"
+require_relative "notifications/notification"
+
+module Notifications
+  class Configuration
+    def call(event_store, command_bus)
+      command_handler = OnNotificationCommands.new(event_store)
+      command_bus.register(CreateNotification, command_handler)
+      command_bus.register(SendNotification, command_handler)
+      command_bus.register(MarkNotificationAsRead, command_handler)
```

Comment:
So this will embrace the system notification as well? For Telegram and Farcaster Integration users can potentially disable this, like Whatsapp does